images/git/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/k8s-prow/alpine:v20200605-44f6c96
+FROM gcr.io/k8s-prow/alpine:latest
```

Comment:
Please use a tagged version. We automatically send PRs to upgrade to the latest tagged version when this changes.

Reply:
That probably means I have to wait until `alpine` is rebuilt with the changes from https://github.com/kubernetes/test-infra/pull/22444, and the auto-PR changing the tag has merged?

Reply:
Hm looks like #22444 broke alpine postsubmit job :/

Reply:
https://github.com/kubernetes/test-infra/pull/22609

Reply:
That issue seemed to be a transient one. Thanks for re-renunning it @fejta!
Thus, this PR here should be good to go now :)